packages/smooth_app/lib/pages/home_page.dart

```diff
@@ -357,6 +358,36 @@ class _HomePageState extends State<HomePage> {
         ),
       );
     }
+
+    Widget _getCards(List<Widget> attributes) {
+      final List<Widget> list = <Widget>[];
+
+      List<void>.generate(
+          maxEntrys, (int index) => list.add(attributes[index]));
+
+      if (attributes.length > list.length) {
+        list.add(
+          ElevatedButton(
```

Comment:
I would create a function that creates that `ElevatedButton`, to be called for the first 6 attributes and for the new "+ 22" button.